### ISSUE TITLE
Removed Swiping: UX vs. Usability

### ISSUE THREAD

**Issue by SecUpwN** [SecUpwN/Android-IMSI-Catcher-Detector#778]
Since merging #776, the swiping navigation is gone. Question directed at @tt3mm and @larsgrefer: Is this change better for following the Android UX rules, or should we add back swiping for better usability? I'm not so sure if people understand that they'll have to access the `Navigation Drawer` each time.

**Comment by tt3mm** [comment]
I think the two together was very confusing. 
Check out the youtube app. As I remember they had a nav drawer but now it's gone, and they have a 4 page swiping. The Gmail app only has nav drawer.
I was sorting my apps this was the usual, if there is a few, frequently used screens, then swiping, otherwise nav drawer.

I would stay with it, because it's a diagnostic application, and I think the users would use 1 screen for longer times. But of course this statement is just a guess and needs some testing.

Also, can you tell me what does the All Current Cell Details is doing? I see it makes a  request to the OpenCellId api, and on success it broadcasts some update signal, but i never seen any use of it.

**Comment by SecUpwN** [comment]
>I think the two together was very confusing. 

I fully agree, @tt3mm. Would you like to specifically tackle the whole re-design in #258? I would then list you as our main `UI/UX Designer` on our [contact page](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Contact) if you like to. Also have a look at our [Style Guide](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Style-Guide).

>I would stay with it, because it's a diagnostic application, and I think the users would use 1 screen for longer times. But of course this statement is just a guess and needs some testing.

You're right. In the long run, we'd like the `Device Details` to move into another place in our app and instead show a much more useful screen like the ones proposed [here](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/201#issuecomment-66763582) and [there](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/201#issuecomment-67660907), but detections for these things need to be implemented first. Are you planning to change our app UX similar like the YouTube app?

**Comment by tt3mm** [comment]
Thanks, I will think about a whole re-design, I already have some ideas, but let me get more involved/understand in the architecture first.

If we stay with the 4-5 main screens then yes, something youtube-like UX could work.

**Comment by SecUpwN** [comment]
>Thanks, I will think about a whole re-design, I already have some ideas, but let me get more involved/understand in the architecture first.

Sounds great! Besides the few hopefully very useful screens we want to display to our users, we'd like our app to be set up correctly with #181 before the user can even start to use it. That way, common errors like a missing download of the OpenCellID database and other things shall be avoided.

>If we stay with the 4-5 main screens then yes, something youtube-like UX could work. 

I will download that app today and have a closer look. Please make sure to always attach a mockup of the proposed changes before coding them. We don't want you to put heartblood into something we'd disagree with (although we're already convinced that you do really awesome work here).
